src/arrayMethodSort.js

```diff
@@ -4,8 +4,19 @@
  * Implement method Sort
  */
 function applyCustomSort() {
-  [].__proto__.sort2 = function(compareFunction) {
-    // write code here
+  [].__proto__.sort2 = function (compareFunction = (a, b) => `${a}` > `${b}`) {
```

Comment:
The default compare function should convert the elements to strings and compare their UTF-16 code units values. Using template literals implicitly calls `toString()` which is correct, but the comparison should be based on the UTF-16 code unit values, so you should use localeCompare or a similar method that compares based on character codes.